pipeline_steps/delivery.py

```diff
@@ -0,0 +1,11 @@
+from datetime import datetime
+import pytz, logging
+from base import PipelineStep
+class PipelineDelivery(PipelineStep):
+    @staticmethod
+    def process_message(message):
+        logging.warning(message.current_step + " has received " + message.name + " " + str(message.id))
+        if pytz.utc.localize(datetime.utcnow()) > message.expiration_time:
+            logging.warning(message.name + " " + str(message.id) + " has expired. It will be dropped.")
```

Comment:
This is already handled by `base.PipelineStep`.  Call `super(PipelineDelivery).process_message(message) before doing things that are specific to this step instead of duplicating that logic here.